src/usr/local/www/firewall_nat_out.php

```diff
@@ -244,7 +244,8 @@
 }
 
 if (is_subsystem_dirty('natconf')) {
-	print_info_box_np(gettext("The NAT configuration has been changed.")."<br />".gettext("You must apply the changes in order for them to take effect."));
+	print_info_box_np(gettext('The NAT configuration has been changed.') . '<br />' .
+					  gettext('You must apply the changes in order for them to take effect.'));
```

Comment:
That is a cut-paste from firewall_nat.php - it has a bunch of tabs that, in an editor with tabs displayed at 4-spaces intervals, will make the 2 gettext() calls line up vertically.
I have never really understood why people have done that, because the resulting vertical line-up depends on the editor display settings used. Personally I would put just 1 tab extra indent from the outer calling function.
This is the same in all 4 files that it is in - I can easily modify them all, but exactly how should it be (is there a definitive style-guide for this?)?